modules/tensor_mechanics/test/include/materials/FunctionIsotropicEigenstrain.h

```diff
@@ -0,0 +1,26 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ComputeEigenstrainBase.h"
+
+class FunctionIsotropicEigenstrain : public ComputeEigenstrainBase
```

Comment:
Slowly subverting the `Compute...` prefix tradition.

Reply:
Yeah, I guess so. I've been working on subverting that so long that I didn't even realize I was doing it.